ksql-rest-app/src/test/java/io/confluent/ksql/api/AuthTest.java

```diff
@@ -35,9 +45,9 @@
 import org.slf4j.LoggerFactory;
 
 @RunWith(CoreApiTestRunner.class)
-public class BasicAuthTest extends ApiTest {
+public class AuthTest extends ApiTest {
```

Comment:
Having a single AuthTest file rather than separate test files for basic auth and the authorization filter seems fine for now but I wonder how this file will scale as we add other auth mechanisms. Don't see a great way to split them apart so I'm inclined to leave this for now, though I am curious if you have thoughts on how to scale these auth tests going forward.

Reply:
Agreed, as per previous comment :)